doc/authentication-backends/Dummy-authentication-module.md

```diff
@@ -0,0 +1,13 @@
+## Overview
+
+The purpose of this module is to make it possible to authenticate a user without
+a need of real authentication. In other words, using this module allows to
```

Comment:
Should read

>[...] a user without the need for real authentication.

Reply:
Why is `the` correct here? I've search the internet for some examples and I'm not quite sure why `the` fits better than `a` here.

Reply:
It was a matter of intuition, when I read the text I just felt `without the need for` would be a more correct approach. I tried looking up in the internet why do I think that is a better option but I also didn't find very clear information about the topic. Ultimately, I just asked two British native speakers and they both agreed with me, so I put it on the review 🤷‍♂

Reply:
Cool, I'm convinced now 😀 Thanks!